src/app/pages/components/tree/tree.component.ts

```diff
@@ -1,12 +1,15 @@
-import { Component } from '@angular/core';
-
+import { Component , OnInit} from '@angular/core';
+import { ViewChild } from '@angular/core';
+import { TreeComponent, TREE_ACTIONS,  ITreeOptions } from 'angular-tree-component';
+import * as _ from 'lodash';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import { InputComponent } from './input.component';
 @Component({
   selector: 'ngx-tree',
   templateUrl: './tree.component.html',
   styleUrls: ['./tree.component.scss'],
 })
-export class TreeComponent {
-
+export class TreeAComponent implements OnInit {
```

Comment:
Was the renaming of the component intentional? If so, why?

Reply:
Because I'm importing tree component module which have the same name as it is.

Reply:
I think it would make more sense to alias the import than to change this component's name
`import { TreeComponent as AngularTreeComponent, TREE_ACTIONS,  ITreeOptions } from 'angular-tree-component';`

Reply:
thanks i will change it